docs/getting-started-guides/centos/centos_manual_config.md

```diff
@@ -196,6 +196,13 @@ for SERVICES in kube-proxy kubelet flanneld docker; do
     systemctl status $SERVICES
 done
 ```
+* Configure kubectl
+
+```shell
+kubectl config set-cluster default-cluster --server=http://centos-master:8080
+kubectl config set-context default-system --cluster=default-cluster --user=default-admin
```

Comment:
I use a lot kubernetes on coreos , so that's how they named their context in docs, but name default-context have more sens.